### PR TITLE
EAR-1455 Fix for properties panel and piping

### PR DESCRIPTION
**Pull request by martyncolmer**
Mop up work for properties panel
Add features identified in review

Update publisher to ignore advanced properties when switched off

https://collaborate2.ons.gov.uk/jira/browse/EAR-1455

### CODE REVIEW

File: src/utils/convertPipes/PlaceholderObjectBuilder.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const TRANSFORM_MAP = {
   Number: { format: FORMAT_NUMBER, transformKey: NUMBER_TRANSFORMATION },
   Date: { format: FORMAT_DATE, transformKey: DATE_TRANSFORMATION },
   DateRange: { format: FORMAT_DATE, transformKey: DATE_TRANSFORMATION },
-  Unit: { format: FORMAT_UNIT, transformKey: NUMBER_TRANSFORMATION },
+  // Unit: { format: FORMAT_UNIT, transformKey: NUMBER_TRANSFORMATION },
```
Comment (ashtuck) [inline src/utils/convertPipes/PlaceholderObjectBuilder.js:24]: Just the commented out code here 👍 .

File: src/utils/convertPipes/PlaceholderObjectBuilder.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+//const { unitConversion } = require("../../constants/units");
 
 const {
   FORMAT_CURRENCY,
   FORMAT_DATE,
   FORMAT_NUMBER,
   NUMBER_TRANSFORMATION,
   DATE_TRANSFORMATION,
-  FORMAT_UNIT,
+  // FORMAT_UNIT,
```
Comment (ashtuck) [inline src/utils/convertPipes/PlaceholderObjectBuilder.js:9]: Another bit of commented code.

File: src/utils/convertPipes/index.test.js [hunk 3]
```diff
@@ -227,23 +227,24 @@ describe("convertPipes", () => {
...
+      //   const html = createPipe({ id: "6" });
+      //   const runnerJSON = createWrapper(
+      //     "{answer6}",
+      //     createTransformation({
+      //       placeholder: "answer6",
+      //       source: "answers",
+      //       argument: "number",
+      //       transform: "format_unit",
+      //     })
+      //   );
```
Comment (ashtuck) [inline src/utils/convertPipes/index.test.js:241]: Commented out code.

File: src/utils/convertPipes/PlaceholderObjectBuilder.js (hunk 1 above)
Comment (martyncolmer) [reply src/utils/convertPipes/PlaceholderObjectBuilder.js:24]: Leaving it in until we know whats happening with unit

File: src/utils/convertPipes/index.test.js (hunk 3 above)
Comment (martyncolmer) [reply src/utils/convertPipes/index.test.js:241]: leaving in until found out about runner

**Review by MattElcock — APPROVED** [review]

**Review by ashtuck — APPROVED** [review]

**Pull request merged**